### ISSUE TITLE
CLI option -import_precurosor_mz

### ISSUE THREAD

**Issue by jj-umn** [compomics/peptide-shaker#158]
Are current CLI options:
  -import_precurosor_mz
  -import_precurosor_mz_ppm
direct replacements for previous options:
  -max_precursor_error
  -max_precursor_error

The description is a little vague on:
  http://compomics.github.io/compomics-utilities/wiki/identificationparameterscli.html#import-filters

Can -import_precurosor_mz value be either ppm or daltons?  

Should  compomics-utilities/src/main/java/com/compomics/util/experiment/identification/parameters_cli/IdentificationParametersInputBean.java  
read the value as a Double rather than an Integer to account for a possible value in Daltons?

        if (commandLine.hasOption(IdentificationParametersCLIParams.IMPORT_PRECURSOR_MZ.id)) {
            String arg = commandLine.getOptionValue(IdentificationParametersCLIParams.IMPORT_PRECURSOR_MZ.id);
            Integer value = new Integer(arg);
            peptideAssumptionFilter.setMaxMzDeviation(value);
        }

**Comment by mvaudel** [comment]
Dear JJ,

Thank you very much for reporting these issues, and especially for debugging and proposing a fix!

> Are current CLI options:
> -import_precurosor_mz
> -import_precurosor_mz_ppm
> direct replacements for previous options:
> -max_precursor_error
> -max_precursor_error

Yes, note that we refactored and grouped all identification parameters in one class. We also changed the handling of the parameters so that you do not need to create an intermediate file and can provide any parameter directly to the tool. If you use a parameters file the format is now open (json), as suggested by @wolski.

> The description is a little vague on:
> http://compomics.github.io/compomics-utilities/wiki/identificationparameterscli.html#import-filters

Apologies, it seems that I jumped a line when writing the description of these parameters. Now corrected, hope that it is clearer now!

> Can -import_precurosor_mz value be either ppm or daltons?

Yes, and this should be specified in the line I forgot originally...

> read the value as a Double rather than an Integer to account for a possible value in Daltons?

Yes again, now corrected, we will release a new version containing this fix very soon!

Best regards,

Marc
